en-US/firefox/features/sync.lang

```diff
@@ -0,0 +1,81 @@
+## URL: https://www-dev.allizom.org/%LOCALE%/firefox/sync/
+
+
+# HTML page title
+;Use Bookmarks, Tabs and Passwords Across Devices | Firefox
+Use Bookmarks, Tabs and Passwords Across Devices | Firefox
+
+
+# HTML page description
+;Bring your Firefox bookmarks, tabs, or passwords wherever you go with Firefox Sync. Share between computers and mobile devices. Get Firefox now!
+Bring your Firefox bookmarks, tabs, or passwords wherever you go with Firefox Sync. Share between computers and mobile devices. Get Firefox now!
+
+
+;Browse uninterrupted with Firefox Sync
+Browse uninterrupted with Firefox Sync
+
+
+# Displayed only to Firefox user with 31 and later, signed in into Firefox Accounts
+;Sync across devices
+Sync across devices
+
+
+;With Firefox on all your devices, you can access bookmarks, tabs and passwords with one easy sign in.
+With Firefox on all your devices, you can access bookmarks, tabs and passwords with one easy sign in.
+
+
+;Sign in to your Firefox Account on mobile or tablet to get your open tabs, bookmarks and passwords anywhere.
+Sign in to your Firefox Account on mobile or tablet to get your open tabs, bookmarks and passwords anywhere.
+
+
+;Sign in or create an account through your Firefox app for seamless browsing of passwords, bookmarks and open tabs.
+Sign in or create an account through your Firefox app for seamless browsing of passwords, bookmarks and open tabs.
+
+
+# Button label displayed to users of Firefox 31 and later, not signed in to Firefox Accounts
+;Create account
+Create account
+
+
+# Text displayed about download buttons for mobile apps (for Firefox 31 and later, signed in to Firefox Accounts)
+;Get the app
+Get the app
+
+
+# Alternate text for the iTunes download button
+;Download on the App Store
+Download on the App Store
+
+
+# Button label displayed to users of Firefox 30 and older
+;Update your Firefox
+Update your Firefox
+
+
+;Features
+Features
+
+
+;Personal
+Personal
+
+
+;That shopping rabbit hole you started on your laptop this morning? Pick up where you left off on your phone tonight. That dinner recipe you discovered at lunchtime? Open it on your kitchen tablet, instantly. Connect your personal devices, securely.
+That shopping rabbit hole you started on your laptop this morning? Pick up where you left off on your phone tonight. That dinner recipe you discovered at lunchtime? Open it on your kitchen tablet, instantly. Connect your personal devices, securely.
+
+
+;Secure
+Secure
+
+
+;Your Firefox Account is the doorway to all your web stuff—we help you keep it safe. Your data is always in your control, unreadable by anyone else, and encrypted with your account password. We protect it and hand you the key.
+Your Firefox Account is the doorway to all your web stuff—we help you keep it safe. Your data is always in your control, unreadable by anyone else, and encrypted with your account password. We protect it and hand you the key.
+
```

Comment:
Firefox Accounts? We can leave this one as is but I am more confused with the variants lately.

Reply:
Firefox Account is correct here, Firefox Accounts is the feature (it would be correct without the word)